### PR TITLE
Global Angular Configuration Check

### LINKED ISSUE #12198: [7 beta] ng new error: "new command requires to be run outside of a project, but a project definition was found", doesn't say where the stray file is
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [X ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions
Angular CLI: 7.0.0-beta.2
Node: 10.10.0
OS: linux x64

(ng new doesn't have other versions yet)

### Repro steps
`ng new newProjecct`

### The log given by the failure
image

### Desired functionality
A new project ;)

### PR DESCRIPTION
**Pull request by clydin**
Fixes #12198

### CODE REVIEW

File: packages/angular/cli/utilities/check-package-manager.ts [hunk 1]
```diff
@@ -1,53 +0,0 @@
-/**
```
Comment (alexeagle) [inline packages/angular/cli/utilities/check-package-manager.ts]: why is this file deleted? is it an unrelated cleanup?

Comment (clydin) [reply packages/angular/cli/utilities/check-package-manager.ts]: Yes. It’s a separate commit.  I can move it out if you want.
